docs/source/uploading_files.rst

```diff
@@ -0,0 +1,115 @@
+==========================
+Uploading Referenced Files
```

Comment:
Perhaps remove name "referenced" as this may create confusion with ReferenceFiles in the data model?